tests/testHelpers.js

```diff
@@ -3,9 +3,9 @@
  * @author  TheJaredWilcurt
  */
 
-const path = require('path');
-const os = require('os');
-const mock = require('mock-fs');
+import { join, dirname } from 'path';
+import { platform as _platform } from 'os';
```

Comment:
```suggestion
import { platform as osPlatform } from 'os';
```
I'd rather use `osPlatform`, or just `platform`, than `_platform`. I associate an underscore prefix as being used on variables in libraries I am consuming to convey they are not for external use (private variables).